File: 1/solution_1.java

public class solution_1{
    static final int threshold = 1000;
    static int sum = 0;

    static int sumMultiples(){
        for(int i = 0;i < threshold;++i){
            if(i % 3 == 0 || i%5 == 0){
                sum += i; 
            } 

        }
        return sum;
    
    }

    public static void main(String args[]){
        System.out.println(sumMultiples());
    }

}

File: 1/solution_1.cs
using System;

namespace ConsoleApp6
{
	public class solution_1 {
		
		public static int sumMultiples(){
			const int threshold = 1000;
			int sum = 0;
			for(int i = 0; i < threshold; i++){
				if(i % 3 == 0 | i % 5 == 0){
					sum += i;
				}
			}
			return sum;
		}
		
		public static void Main(string[] args){
			Console.WriteLine(solution_1.sumMultiples());
		}
	}
}